### ISSUE TITLE
[Enhancement request] Print `printer-uri` when ippserver starts

### ISSUE THREAD

**Issue by KurtPfeifle** [istopwg/ippsample#63]
**TL;DR:** *`ippserver`* should print *`printer-uri=...`* upon startup, even if no *`-vv`* flag is given.

----

Current setup:

1. When *`ippserver`* starts without any flag (other than *`"name"`*), it prints one line for each client connection, like so:
    ```
    [Client 1] Client closed connection.
    [Client 2] Client closed connection.
    [....]
    ```
    However, it does not print the *`printer-uri`*. How should a newbie user know, what this ominous "uri" is which she should point her IPP client connections to?

1. When *`ippserver`* starts with flag *`-v`*, it is a bit more verbose, like so:
    ```
    Using default configuration with a single printer.
    Using default data directory "/tmp/ippserver.951".
    Using default spool directory "/tmp/ippserver.951".
    Ignore Avahi state 2.
    Using default listeners for raspi3.fritz.box:22222.
    [Client 1] Accepted connection from "192.168.76.34".
    [Client 1] POST /ipp/print
    [Client 1] Continue
    [Client 1] Get-Printer-Attributes successful-ok
    [Client 1] OK
    [Client 1] Client closed connection.
    [Client 1] Closing connection from "192.168.76.34".
    [Client 2] Accepted connection from "192.168.76.34".
    [Client 2] POST /ipp/print
    [Client 2] Continue
    [Client 2] Get-Printer-Attributes successful-ok
    [Client 2] OK
    [Client 2] Client closed connection.
    [Client 2] Closing connection from "192.168.76.34".
    [....]
    ```
    At least, now the port number is printed. But a newbie user will still not know what *`printer-uri`* she should use for *`ipptool`* or other clients to connect to the IPP server...

1. Only when *`ippserver`* starts with double-flag *`-vv`*, it is verbose enough in order to include the *`printer-uri`* info in its *stdout* messages:
    ```
    Using default configuration with a single printer.
    Using default data directory "/tmp/ippserver.1123".
    Using default spool directory "/tmp/ippserver.1123".
    Ignore Avahi state 2.
    Using default listeners for raspi3.fritz.box:22222.
    serverCreatePrinter(resource="/ipp/print", name="A", location="", make="Test", model="Printer", icon="(null)", docformats="application/pdf,image/jpeg,image/pwg-raster", ppm=0, ppm_color=0, duplex=0, pin=0, attrs=(nil), command="(null)", device_uri="(null)", proxy_user="(null)", strings=(nil))
    Using ppm=0
    Using ppm_color=0
    [Printer A] printer-more-info="https://raspi3.fritz.box:22222/ipp/print"
    [Printer A] printer-supply-info-uri="https://raspi3.fritz.box:22222/ipp/print/supplies"
    [Printer A] printer-uri="ipp://raspi3.fritz.box:22222/ipp/print"
    [....]
    ```
    However, while now the info is present in the output, it has scrolled by so fast, that you hardly can recognize it, because it is burried at the top of 264 lines of initial output text.

It would be nice to see a line similar to   
  *`[Printer A] printer-uri="ipp://raspi3.fritz.box:22222/ipp/print"`*    
for every startup output, regardless of the *`-v`* or *`-vv`* parameters given. This little bit of info is probably the most important single piece of them all, because it is required to know for each single client to even start a connection.

**Comment by michaelrsweet** [comment]
Reasonable request, will work on making that change after my next merge of the CUPS code...

**Comment by michaelrsweet** [comment]
OK, I updated the code to log the printer URI as an info-level message, so you'll see the printer URI with a single "-v".

I also updated the code to log the "client closed connection" message as info instead of error, so you won't see them without at least a single "-v".

[master ba2ef14] Log printer URI as an INFO message.
